cluster/saltbase/salt/kube-addons/init.sls

```diff
@@ -101,13 +101,17 @@ addon-dir-create:
     - group: root
     - mode: 755
 
-{% if grains['os_family'] == 'RedHat' %}
+{% if pillar.get('is_systemd') %}
 
-/usr/lib/systemd/system/kube-addons.service:
+{{ pillar.get('systemd_system_path') }}/kube-addons.service:
```

Comment:
i think the same holds here, can kube-addons service.running just have a require on this file existing?